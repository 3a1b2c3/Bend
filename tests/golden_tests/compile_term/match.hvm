switch x = (+ 0 1) {
  0: λt λf t
  _: λ* λt λf f // The term to hvmc function assumes the pred variable is already detached from the match
}