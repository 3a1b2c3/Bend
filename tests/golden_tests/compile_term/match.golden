a
& <#0 <#1 ? ((b (* b)) (* (* (c c)))) a>> ~ #1